easybuild/easyblocks/s/superlu.py

```diff
@@ -84,22 +89,9 @@ def configure_step(self):
                 # 'Intel' -> For older versions of mkl 32 and 64 bit
                 self.cfg.update('configopts', '-DBLA_VENDOR="Intel"')
 
-        elif toolchain_blas in ['ACML', 'ATLAS']:
+        elif toolchain_blas in ['ACML', 'ATLAS', 'OpenBLAS', 'FlexiBLAS']:
```

Comment:
This approach removes backwards compatibility, see my comment for a version that retains it.

Reply:
Yeah, that was my concern as well. The site did not update, so I only saw that after I posted my comments.